src/RVRP_data.jl

```diff
@@ -70,7 +80,7 @@ mutable struct VehicleCategory
     index::Int # Not given in JSON
     fixed_cost::Float64
     unit_pricing::UnitPricing
-    capacity::Float64
+    compartment_capacities::Vector{Float64} # the santard case is to have a single compartment
```

Comment:
I like introducing compartments, it's something we may really need and an advantage for Coluna :thumbsup: